tests/subsys/net/lib/lwm2m_fota_utils/src/firmware.c

```diff
@@ -55,25 +55,15 @@ struct firmware_object {
 };
 
 static struct firmware_object test_object[2];
-static uint8_t *stream_buf;
-static size_t stream_buf_len;
 static fota_download_callback_t firmware_fota_download_cb;
 
 static struct settings_handler *handler;
 static int fota_download_ret_val;
-static int boot_img_num;
+static bool boot_sheduled;
```

Comment:
```suggestion
static bool boot_scheduled;
```
This file seems have few similar typos